### PR TITLE
Add possibility to override the conn type for Druid

### PR DESCRIPTION
**Pull request by olegmayko**
Minor fix, which allows to use the schema which are specified in theschema rather than `http` as default. In the same time it doesn't changethe logic as any conn_type can be selected. Intuitevely it's expectedthat anything specified in `schema` field will actually take precedencein the building the desired url.

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by olegmayko** [comment]
similar issue https://github.com/apache/airflow/issues/27581

**Review by dirrao — COMMENTED** [review]
Nice. Can you add the test case for the same?

**Comment by olegmayko** [comment]
Hi, can you please validate it, I'd like to remove the step from building docker image, which replace the upstream file with the fixed one. Thank you

**Review by potiuk — APPROVED** [review]

File: airflow/providers/apache/druid/hooks/druid.py [hunk 1]
```diff
@@ -86,7 +86,10 @@ def get_conn_url(self, ingestion_type: IngestionType = IngestionType.BATCH) -> s
         """Get Druid connection url."""
         host = self.conn.host
         port = self.conn.port
-        conn_type = self.conn.conn_type or "http"
+        if self.conn.schema:
+            conn_type = self.conn.schema
+        else:
+            conn_type = self.conn.conn_type or "http"
```
Comment (amoghrajesh) [inline airflow/providers/apache/druid/hooks/druid.py:92]: Can we define "http" as a constant that denoted the default connection type?

**Comment by olegmayko** [comment]
hi, can we reopen it, ready to fix it. thx

**Pull request closed without merging**